sdk/apimanagement/api-management-custom-widgets-tools/test/index.spec.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  ON_CHANGE_MESSAGE_KEY,
+  EDITOR_DATA_KEY,
+  onChangeWithOrigin,
+} from "../src"
+import { assert } from "chai";
+import sinon from "sinon";
+import valuesUrl from "./valuesUrl.json";
+
+const searchParams = new URLSearchParams("https://localhost:3000");
+searchParams.set(EDITOR_DATA_KEY, JSON.stringify(valuesUrl));
+
+/*
+describe("getEditorData", () => {
+  let editorData: TEditorData<typeof valuesDefault>;
+
+  it("runs", () => {
```

Comment:
your tests depend on previous tests. It is recommended to make each test self-contained so that any test can successfully run in isolation. Maintaining state across tests is usually a source of test bugs and flaky issues.